src/main/resources/project/WebLogicCreateConfigForm.xml

```diff
@@ -78,11 +87,23 @@ limitations under the License.
         </documentation>
     </formElement>
     <formElement>
-        <type>entry</type>
-        <label>Debug level:</label>
+        <type>select</type>
+        <label>Debug level</label>
```

Comment:
Nope, Debug Level: will be right way of calling the parameter. Because best practices.